incubator/scylladb/tiller-clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: tiller-clusterrolebinding
+subjects:
+- kind: ServiceAccount
+  name: tiller
```

Comment:
The chart expects the `tiller` service account to exist here. The chart should rather create a `ServiceAccount` and use that one.

Reply:
As I stated before, new to k8s/helm. I'll defintely look into it.